packages/amf/amf.0.1.2/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "2.0"
+synopsis: "Parser/serializer for Adobe's Action Message Format"
+maintainer: "Brian Caine <brian.d.caine@gmail.com>"
+authors: "Brian Caine <brian.d.caine@gmail.com>"
+license: "LGPL with OCaml linking exception"
+homepage: "https://briancaine.github.io/ocaml-amf/"
+bug-reports: "https://github.com/briancaine/ocaml-amf/issues"
+depends: [
+  "ocaml" {>= "4.01"}
+  "dune" {build & >= "1.0"}
+  "ppx_deriving" {build}
+  "ppx_sexp_conv" {build}
+  "base-threads"
+  "core" {>= "v0.9.1"}
+  "stdint"
+  "sexplib"
+  "bisect_ppx" {build}
+  "ounit" {build}
+]
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  [make "test"] {with-test}
```

Comment:
```suggestion
  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
```
There doesn't seem to be any reason to require make here, furthermore calling `dune runtest` without its `-p` argument is going to cause breakage in the future as it unables some `-warn-error` to be activated